README.md

```diff
@@ -57,17 +50,26 @@ To start Express, run the following from the root directory:
 
 This will run the express server on `http://localhost:3000`
 
-## Run Client-Side Tests
+### 6. Local HTTP-Proxy to Facet Service
+
 
-Run the following from the `spa` directory:
+The URL for the back end Azul Facet Service is configurable from the environment property ```BW_DATA_URL```. This can be set to the url of the server running the back end in the Gruntfile to run locally or on the boardwalk node.js server.
 
-	npm test
-	
-This will run once through test-suite using PhantomJS. To run through PhantomJS with a file watcher, you can use:
+If this value is not set, the Boardwalk Angular client will assume the Facet Server is at the same url that served the front end, and make a relative request to /api/v1 to retrieve the facets and other configuration. 
 
-    npm test:headless
+To support this, on localhost, run the proxy.json file with `node proxy.json` This will start a proxy server at port 3001. ```proxy.conf.json``` in ```/spa``` is configured to forward requests to /api to the local node.js server at port 3000 and to send /api/v1 to requests to the proxy listening at port 3001.
```

Comment:
1. That should be `node proxy.js`, not "...proxy.js**on**", shouldn't it?
2. Could you instead add a script in package.json to do this? I haven't run the UI in a while, and so simple things like running node, even though it seems pretty obvious how to do it (although maybe the next step is my fault, so maybe not), that it would just be nice to say `npm run proxy` or something like that. Just a suggestion; if you prefer it this way, then that's fine.
3. When I try running `node proxy.js`, I get the error below. Does http-proxy need to be added as a dev dependency to package.json?

```
node proxy.js
module.js:478
    throw err;
    ^

Error: Cannot find module 'http-proxy'
    at Function.Module._resolveFilename (module.js:476:15)
    at Function.Module._load (module.js:424:25)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/charlesoverbeck/git/DataBiosphere/cgp-boardwalk/proxy.js:9:17)
    at Module._compile (module.js:577:32)
```